lib/gitlab/configuration.rb

```diff
@@ -32,6 +32,7 @@ def options
     def reset
       self.endpoint       = ENV['GITLAB_API_ENDPOINT']
       self.private_token  = ENV['GITLAB_API_PRIVATE_TOKEN']
+      self.auth_token     = ENV['GITLAB_API_AUTH_TOKEN']
```

Comment:
Yes, I want to avoid confusion with existing `private_token`.
Another option would be to check the length of supplied token and auto-detect.